test/model.test.js

```diff
@@ -5540,25 +5540,26 @@ describe('Model', function() {
 
         return co(function*() {
           const createdUser = yield User.create({ name: 'Hafez' });
-          let threw = false;
+
+          let err;
+
           try {
```

Comment:
Better to just do `const err = yield User.bulkWrite(ops).then(() => null, err => err);` so you don't need to write out the whole `try/catch` block.

Reply:
More on [golang-in-JS style error handling](http://thecodebarbarian.com/async-await-error-handling-in-javascript.html#golang-in-js)

Reply:
Yes, that looks much cleaner. I just opened another PR with that modification.